dast/__main__.py

```diff
@@ -29,21 +39,80 @@ def main(then_path: str, now_path: str, verbose: bool = True):
         return None, None
     with _open_utf(then_path) as f_then:
         then_ast = ast.parse(f_then.read())
-        prettify(then_ast)
 
     with _open_utf(now_path) as f_now:
         now_ast = ast.parse(f_now.read())
-        prettify(now_ast)
 
     # This also includes end_lineno and end_col_offset
     ignored_props = {"type_ignores", "type_comment", "col_offset", "lineno"}
     callback = lambda _, path: any(path.endswith(prop) for prop in ignored_props)
-    diff = DeepDiff(then_ast, now_ast, exclude_obj_callback=callback)
+    diff = DeepDiff(then_ast, now_ast, ignore_order=True, exclude_obj_callback=callback)
     if diff and verbose:
-        print(f"diff for file {now_path}")
-        print(diff)
+        print_diff(diff, now_path, then_ast, now_ast)
+
+    return now_ast, then_ast, diff
+
+def describe_node(node):
+    if isinstance(node, ast.Module):
+        return "module body"
+    if isinstance(node, ast.If):
+        return f"if statement"
+    if isinstance(node, ast.AugAssign):
+        return f"augmented assignment"
+    if isinstance(node, ast.Call):
+        return f"call to function {ast.unparse(node.func)}"
+    if isinstance(node, ast.keyword):
+        return "keyword"
+    if isinstance(node, ast.Name):
+        return "variable"
+    if isinstance(node, ast.For):
+        return "for loop"
+    if isinstance(node, ast.FunctionDef):
+        return f"function definition '{node.name}'"
+
+    return str(node.__class__)
+
+def describe_change(then, now, change_type, path):
+    msg = ""
+
+    is_added = True
+    parent = describe_node(eval(path.replace("root", "then").rpartition(".")[0]))
+    if change_type == "iterable_item_removed":
+        node = describe_node(eval(path.replace("root", "then")))
+        msg += f"{node} removed from {parent}"
+        is_added = False
+    elif change_type in ["type_changes", "values_changed"]:
+        msg += f"Item changed in {parent}"
+    elif change_type == "iterable_item_added":
+        node = describe_node(eval(path.replace("root", "now")))
+        msg += f"{node} added to {parent}"
+        is_added = True
+    else:
+        return change_type + " unknown", None
+
+    return msg, is_added
+
+
+def print_diff(diff, now_path, then, now):
+    print(f"diff --git a/{now_path} b/{now_path}")
+    print(f"--- a/{now_path}")
+    print(f"+++ b/{now_path}")
+    for change_type, changes in diff.items():
+        for path, change in changes.items():
+            description, is_added = describe_change(then, now, change_type, path)
+            print(f"{bcolors.OKCYAN}@@ -{0},{0} +{0},{0} @@{bcolors.ENDC} {description}")
```

Comment:
This should not be the line and column numbers instead of 0s?

Reply:
Yup!